catalog/registry/catalog-registry-api-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,113 @@
+    <reference id="securityManager" interface="ddf.security.service.SecurityManager"/>
+
+    <reference id="xmlParser" interface="org.codice.ddf.parser.Parser" filter="(id=xml)"
+               availability="mandatory"/>
+
+    <bean id="metacardTypes" class="ddf.catalog.util.impl.SortedServiceList"/>
+    <reference-list id="metacardTypeList" interface="ddf.catalog.data.MetacardType">
+        <reference-listener ref="metacardTypes" bind-method="bindPlugin"
+                            unbind-method="unbindPlugin"/>
+    </reference-list>
+
+    <!-- MetacardTransformer Manager -->
+    <bean id="metacardTransformerManager"
+          class="org.codice.ddf.spatial.ogc.csw.catalog.common.transformer.TransformerManager">
+        <argument ref="metacardTransformers"/>
+    </bean>
+
+    <bean id="metacardTransformers" class="ddf.catalog.util.impl.SortedServiceReferenceList"/>
+    <reference-list id="metacardTransformerList" member-type="service-reference"
+                    interface="ddf.catalog.transform.MetacardTransformer">
+        <reference-listener bind-method="bindService" unbind-method="unbindService"
+                            ref="metacardTransformers"/>
+    </reference-list>
+
+    <bean id="cswTransactionWriter" class="org.codice.ddf.spatial.ogc.csw.catalog.common.source.writer.CswTransactionRequestWriter">
+        <argument ref="cswTransformProvider"/>
+    </bean>
+
+    <cm:managed-service-factory
+            id="ddf.catalog.registry.api.impl.RegistryStoreImpl.id"
+            factory-pid="Csw_Registry_Store">
+        <interfaces>
+            <value>ddf.catalog.registry.api.RegistryStore</value>
+            <value>ddf.catalog.source.CatalogStore</value>
+            <value>ddf.catalog.source.FederatedSource</value>
+        </interfaces>
+        <cm:managed-component class="ddf.catalog.registry.api.impl.RegistryStoreImpl"
+                              init-method="init" destroy-method="destroy">
+            <property name="context" ref="blueprintBundleContext"/>
+            <property name="filterBuilder" ref="filterBuilder"/>
+            <property name="filterAdapter" ref="filterAdapter"/>
+            <property name="cswUrl" value=""/>
+            <property name="id" value=""/>
+            <property name="username" value=""/>
+            <property name="password" value=""/>
+            <property name="disableCnCheck" value="false"/>
+            <property name="coordinateOrder" value="LON_LAT"/>
+            <property name="usePosList" value="false"/>
+            <property name="metacardMappings">
+                <list/>
+            </property>
+            <property name="pollInterval" value="5"/>
+            <property name="resourceReader" ref="resourceReader"/>
+            <property name="outputSchema" value="http://www.opengis.net/cat/wrs/1.0"/>
+            <property name="queryTypeName" value=""/>
```

Comment:
These are required properties that the CswSource implementation needs in order to correctly build GetRecords Requests.